volume/validate_test.go

```diff
@@ -31,13 +31,9 @@ func TestValidateMount(t *testing.T) {
 
 		{mount.Mount{Type: mount.TypeBind, Source: testDir, Target: testDestinationPath}, nil},
 		{mount.Mount{Type: "invalid", Target: testDestinationPath}, errors.New("mount type unknown")},
+		{mount.Mount{Type: mount.TypeBind, Source: testSourcePath, Target: testDestinationPath}, errBindNotExist},
 	}
-	if runtime.GOOS == "windows" {
```

Comment:
Not sure why this was skipped on Linux; this was done as part of https://github.com/moby/moby/commit/e89b6e8c2d2c36c43f22aeaf2a885646c2994051#diff-6f41eadf3d19741b898f4be682e5d670R35 (https://github.com/moby/moby/pull/34258), but that also added this comment; https://github.com/moby/moby/blame/c492e76d13a61b13815475d1a6393ab1da1db5c8/volume/linux_parser.go#L43-L46

/cc @simonferquel